### PR TITLE
Load data dialog

### PR DESCRIPTION
**Pull request by WilliamRichards2017**

### CODE REVIEW

**Comment by anderspitman** [comment]
quarantest build successful. Demo link at
http://4a048679.quarantest.iobio.io

**Comment by anderspitman** [comment]
quarantest build successful. Demo link at
http://6120d470.quarantest.iobio.io

**Comment by WilliamRichards2017** [comment]
To test, try opening and closing the files dialog multiple times.  Try from the load your data button, or the navbar dropdown.

File: .idea/gene.iobio.vue.iml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/.tmp" />
+      <excludeFolder url="file://$MODULE_DIR$/temp" />
+      <excludeFolder url="file://$MODULE_DIR$/tmp" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (adityaekawade) [inline .idea/gene.iobio.vue.iml:12]: @WilliamRichards2017  Looks like IDE files got committed. Can you add the .idea folder to `.gitignore` in your next commit?

**Review by adityaekawade — APPROVED** [review]
Tested the functionality on quarantest. Works as expected.

File: .idea/gene.iobio.vue.iml (hunk 1 above)
Comment (WilliamRichards2017) [reply .idea/gene.iobio.vue.iml]: Done!

**Comment by anderspitman** [comment]
quarantest build successful. Demo link at
http://3e573ef9.quarantest.iobio.io

**Pull request merged**